externals/nghttp2/src/makeHook.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+make clean
+make
+make install
```

Comment:
It's been configured to install in the same directory where it's being built in order to be able to use the CURL's "--with-nghttp2=<path>" building option. That groups up all the generated artifacts in a lib/ directory which is required to build it.